domains/daudix.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "daudix",
+        "email": "ddaudix@gmail.com"
+    },
+    "record": {
+        "CNAME": "daudix.codeberg.page"
```

Comment:
I cannot verify the site `daudix.codeberg.page` because it redirects to the is-a.dev domain (it's probably because you already set the CNAME in your server)